pkg/storage/couchdb/couchdbstore.go

```diff
@@ -346,6 +364,38 @@ func (i *couchDBResultsIterator) Value() ([]byte, error) {
 	return value, nil
 }
 
+func (c *CouchDBStore) getAllKeyValuePairs(rows *kivik.Rows) (map[string][]byte, error) {
+	allKeyValuePairs := make(map[string][]byte)
+
+	nextOK := rows.Next()
+	for nextOK {
+		key := rows.Key()
+		// The returned key is a raw JSON string. It needs to be unescaped:
+		key, err := c.unquote(key)
```

Comment:
when can an "unquote" function ever fail?

Reply:
ok, saw godoc for strconv.Unquote... seems like an unnecessary feature for this use case

Reply:
imagine if strings.TrimPrefix() also started failing... makes no sense. Oh well.